Formula/ice.rb

```diff
@@ -58,6 +58,7 @@ def install
     args = %W[
       prefix=#{prefix}
       USR_DIR_INSTALL=yes
+      SLICE_DIR_SYMLINK=yes
```

Comment:
Think more along the lines of what shells do. 

eg.
* bash completions go in `/usr/local/share/bash-completion`
* fish functions/completions go in `/usr/local/share/fish`
* zsh functions/completions go in `/usr/local/share/zsh`

Ice's slice files fall under the same idea. We don't "need" it to be a folder, its just a convenience on other platforms that allow this (rpm, apt, etc.). Since there are no other brew formula (that I know of) which install slice files, then just creating this as a symlink is perfectly sufficient.